client/src/components/controls/EventChart.vue

```diff
@@ -149,63 +146,44 @@ export default {
       }
       canvas.width = this.clientWidth;
       canvas.height = bars.slice(-1)[0].top + 15;
-      const overflow = 1.1; // CHANGE to > ~1.05 if you want overlapping or not keyframes
-      let selectedBar = null;
+      const muteOpacity = '30'; // Hex string: how much to mute regular colors
```

Comment:
```suggestion
      const muteOpacity = '30'; // Hex string: how much to mute regular colors: '#RRGGBB[AA]'
```
I know myself and even knowing exactly what is being done because I've done it quite a bit before, my head keeps going 30% opacity.  I know I'd adjust that number to like 100 and wonder what's going on, even with the comment right there  Just a suggestion, not really necessary.